ci-operator/config/red-hat-storage/ocs-operator/red-hat-storage-ocs-operator-main__latest-upstream.yaml

```diff
@@ -0,0 +1,94 @@
+base_images:
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.9
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: FROM ceph/daemon-base:latest-master
+  to: ceph-master
+- dockerfile_literal: FROM rook/ceph:master
+  to: rook-master
+- dockerfile_literal: FROM quay.io/ocs-dev/noobaa-core:master
+  to: noobaa-core-master
+- dockerfile_literal: FROM quay.io/ocs-dev/noobaa-operator:master
+  to: noobaa-operator-master
+- dockerfile_path: build/Dockerfile.build
+  from: os
+  inputs:
+    root:
+      as:
+      - golang:1.16
+  to: ocs-operator-master
+- context_dir: openshift-ci/
+  dockerfile_path: index.Dockerfile
+  from: operator-index
+  to: noobaa-index
+operator:
+  bundles:
+  - as: latest-upstream
+    base_index: noobaa-index
+    context_dir: deploy/
+    dockerfile_path: Dockerfile
+  substitutions:
+  - pullspec: quay.io/ocs-dev/ocs-operator:.*
+    with: ocs-operator-master
+  - pullspec: ceph/daemon-base:.*
+    with: ceph-master
+  - pullspec: rook/ceph:.*
+    with: rook-master
+  - pullspec: noobaa/noobaa-core:.*
+    with: noobaa-core-master
+  - pullspec: noobaa/noobaa-operator:.*
+    with: noobaa-operator-master
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
+- as: ocs-operator-bundle-e2e-aws
```

Comment:
Of note, this test name was changed to be the same name as the test that runs for pull requests (defined in `red-hat-storage-ocs-operator-main.yaml`).

It isn't beneficial to make this a special name because each test runs in a different test namespace. Adding `-master-` or `-latest-upstream-` into this name really only makes the name appear longer on the GitHub status, and `-latest-upstream-` is already added because of the `variant` information.